lib/handlers/get.js

```diff
@@ -176,7 +176,7 @@ function globHandler (req, res, next) {
       var data = $rdf.serialize(
         undefined,
         globGraph,
-        null,
+        filename,
```

Comment:
My point was not regarding the asterisk. It was regarding the lack of a URI scheme